### DISCUSSION TITLE
Issue with plugin install

### DISCUSSION THREAD

**Discussion by vitmax1** [community/community#108915] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

`"SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cg80884_octo.publipresse_forms_records' doesn't exist (SQL: select count(*) as aggregate from `publipresse_forms_records` where `unread` = 1)" on line 760 of /home/c/cg80884/vxprt/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php`

when i try to install your plugin i get an error. How to fix it?

**Comment by CoreTiQ** [comment]
php artisan migrate
